### PR TITLE
General refactor

### PR DESCRIPTION
**Pull request by EotronRM**

### CODE REVIEW

File: e2e/pages/login.po.ts [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
 
-  async tryToLogIn() {
-    await element(by.css('[name="user"]')).sendKeys('nestor.otondo@fundacion-jala.org');
-    await element(by.css('[name="password"]')).sendKeys('Zeus2Deus');
-    await element(by.css('#login')).click();
-    return new Dashboard();
-  }
+    async getTitlePage() {
+        await CommonActions.waitVisibility(element(by.css('h1')))
+        return element(by.css('h1')).getText();
```
Comment (carledriss) [inline e2e/pages/login.po.ts]: I think you can create a getText function on CommonActions class

File: e2e/utils/CommonActions.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        await browser.wait(ExpectedConditions.visibilityOf(element), TIMEOUT);
+        await element.clear();
+        await element.sendKeys(value);
+    }
+
+    static async waitVisibility(element: ElementFinder) {
+        await browser.wait(ExpectedConditions.visibilityOf(element), TIMEOUT);
+    }
+
+    static async waitVisibilityandSubmit(element: ElementFinder) {
```
Comment (carledriss) [inline e2e/utils/CommonActions.ts]: this method should be called submit

File: protractor.conf.ts [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
-  }
+    framework: 'jasmine',
+    capabilities: {
+        browserName: 'chrome',
+    },
+    specs: [
+        './e2e/specs/dashboard.spec.js',
+    ],
+    seleniumAddress: 'http://localhost:4444/wd/hub',
+    SELENIUM_PROMISE_MANAGER: false,
```
Comment (carledriss) [inline protractor.conf.ts:12]: great! 👍

File: e2e/utils/CommonActions.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import {browser, ElementFinder, ExpectedConditions} from 'protractor';
+
+const TIMEOUT = 30000;
+
+export class CommonActions {
```
Comment (carledriss) [inline e2e/utils/CommonActions.ts:11]: Don't forget to add documentation.

**Review by carledriss — DISMISSED** [review]

**Review by abelgustavo — APPROVED** [review]

**Pull request merged**